src/debug/debug_disasm.cpp

```diff
@@ -311,28 +311,28 @@ static char const *second[] = {
   "setl %Eb",         "setge %Eb",       "setle %Eb",      "setg %Eb",
 /* a */
   "push fs",          "pop fs",          "cpuid",          "bt %Ev,%Gv",
-  "shld %Ev,%Gv,%Ib", "shld %Ev,%Gv,cl", 0,                0,
-  "push gs",          "pop gs",          0,                "bts %Ev,%Gv",
-  "shrd %Ev,%Gv,%Ib", "shrd %Ev,%Gv,cl", 0,                "imul %Gv,%Ev",
+  "shld %Ev,%Gv,%Ib", "shld %Ev,%Gv,cl", nullptr,                nullptr,
+  "push gs",          "pop gs",          nullptr,                "bts %Ev,%Gv",
+  "shrd %Ev,%Gv,%Ib", "shrd %Ev,%Gv,cl", nullptr,                "imul %Gv,%Ev",
```

Comment:
Indentation seems to be off with the rest.

Reply:
This looked like this before:

<img width="654" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/eefaf54f-7ef9-4584-9ec8-9752dbb27fa9">

Now it looks like this; the tabular formatting is lost:

<img width="748" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/a7e0f77f-03ce-45c8-90b3-c8d045420398">